code/modules/mob/living/carbon/xenomorph/Castes/Predalien.dm

```diff
@@ -3,8 +3,8 @@
 	display_name = "Abomination"
 	upgrade_name = ""
 	caste_type_path = /mob/living/carbon/Xenomorph/Predalien
-	tier = 1
-	upgrade = 0
+	tier = XENO_TIER_ONE
+	upgrade = XENO_UPGRADE_INVALID
```

Comment:
Catched.

Reply:
They can't upgrade so i changed them to this for consistency